.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug report
+description: Create a report to help us improve
+title: '[issue summary] Please verify version before submitting new issue'
+labels:
+  - 'Type: Bug'
+  - not-reviewed
+  - auro-badge
```

Comment:
Watch that copy and paste... there are multiple references to `badge` in this repo.